frontend/src/app/projects/models/model-source/t4c/manage-t4c-model/manage-t4c-model.component.html

```diff
@@ -71,6 +71,15 @@
           <mat-icon class="mat-icon-position">arrow_back</mat-icon>
           Choose primary source
         </button>
+        <button
+          *ngIf="!asStepper"
+          (click)="unlinkT4CModel()"
+          mat-flat-button
+          color="warn"
+          type="button"
+        >
+          Unlink
+        </button>
```

Comment:
```suggestion
        <button
          *ngIf="editing"
          (click)="unlinkT4CModel()"
          mat-flat-button
          color="warn"
          type="button"
        >
          Unlink
        </button>
```
We are also able to create a new t4c instance on the `Configure model sources` page, where `asStepper` is false. I think using `editing` instead works well, since it should only be true if we are actually editing an existing t4c model.